Makefile.in

```diff
@@ -318,13 +318,20 @@ distprep: catman-do
 	$(AUTORECONF)
 	-rm -rf autom4te.cache
 
-install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf host-key check-config
+install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-iron-files install-sysconf host-key check-config
```

Comment:
We could delete this line since I'm just calling `make install-iron-files` directly at present in the packaging scripts. I added it here in case someone forgets and just does a `make install`.